sdk/highlight-go/otel.go

```diff
@@ -167,6 +168,12 @@ func StartOTLP() (*OTLP, error) {
 		return nil, err
 	}
 
+	propagator := propagation.NewCompositeTextMapPropagator(
+		propagation.TraceContext{},
+		propagation.Baggage{},
+	)
+	otel.SetTextMapPropagator(propagator)
```

Comment:
This was required for span propagation to work from the client to the server.

Reply:
could we check that our other SDKs configure this too? for customers on node.js backend, ruby, etc

Reply:
Yeah, we should set this up so that we are sending/receiving the necessary data for propagation in all our SDKs. I can file a ticket to follow up for each of them.

Reply:
https://linear.app/highlight/issue/HIG-4771/configure-otel-propagation-in-all-sdks